Cargo.toml

```diff
@@ -28,9 +28,9 @@ overflow-checks = true
 #backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cosmwasm-std = { version = "1.3.3" }
-cw-storage-plus = { version = "1.1.0" }
-provwasm-std = { version = "2.0.0" }
+cosmwasm-std = { version = "2.1.0" }
```

Comment:
Latest version is `2.1.1`, i'm assuming that would be compatible with these changes if you just wanted to upgrade to the latest